frontend/src/components/navbar.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { ChangeEvent } from 'react';
+import Link from 'next/link';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faChartSimple, faChartPie, faChartLine } from '@fortawesome/free-solid-svg-icons';
+
+function Navbar(){
+    return <div className='navigation'>
+        <h1>Perfguardian</h1>
+        <div className="logoContainer">
+        <svg className='navigation_logo' width="421" height="73" viewBox="0 0 421 73" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
<img className='logo' src="/perfguardian-text-and-logo.svg" alt='perfguardian-text-and-logo' />
 rajouter un filtre pour le changement de couleur => darkmode